app/spdb/build.sbt

```diff
@@ -502,6 +503,30 @@ def gnodb(version: Version) = AppConfig(
   )
 ) extending List(odbproduction(version), gnodb_credentials(version))
 
+// GN AGS ODB: hbfauxodb-lv1 / gnauxodb
+def gnagsodb(version: Version) = AppConfig(
```

Comment:
Yes, the frameworklevel has to be set to 100 for the datamanager to run, and that must be manually done.